documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -1,11 +1,19 @@
 // * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="creating-migration-plan-2-6-3_{context}"]
-ifdef::provider[]
-= Creating a migration plan starting with a source provider
+[id="creating-migration-plan-2-8_{context}"]
+= Creating a migration plan
 
-You can create a migration plan based on a source provider, starting on the *Plans for virtualization* page. Note the specific options for migrations from VMware or {rhv-short} providers.
+You can create a migration plan by using the {ocp} web console to specify a source provider, the virtual machines (VMs) you want to migrate, and other plan details.
+
+[WARNING]
+====
+Virtual machines with guest initiated storage connections, such as Internet Small Computer Systems Interface (iSCSI) connections or Network File System (NFS) mounts, are not handled by {project-short} and could require additional planning before or reconfiguration after the migration.
+
+This is to ensure that no issues arise due to the addition or newly migrated VM accessing this storage.
```

Comment:
It’s not clear to me what “due to the addition or newly migrated VM” means.  What’s the “addition VM”?  Does this mean this:

This ensures that no issues arise when a newly added VM or a newly migrated VM accesses this storage.

:-) Don't let me change the technical meaning if this is not what you are trying to say.

Reply:
@fabiand This change looks OK to me. WDYT?

Reply:
Nunzy, thanks for finding this, but you do change the technical meaning, what about

> This is to ensure that no concurrent access to the storage arises after VM migration.

Reply:
@nunzy1 After chatting a bit, Fabian and I settled on "This prevents concurrent disk access to the storage the guest points to."